language/stdlib/modules/lcs.move

```diff
@@ -0,0 +1,7 @@
+// Utility for serializing a Move type to its binary LCS representation
+
+address 0x0:
+module LCS {
+    // Return the LCS serialization of this Move value
+    native public fun serialize<Value>(t: &Value): vector<u8>;
```

Comment:
Nit. Is `Value` the best name here? Sometimes we refer to non-resources as 'value's (but the parameter here could be a resource). I think `T` might just be better here? Thoughts?

Reply:
Hmm. I'm also reluctant to use `T` because of the  overlap with the  `resource struct T` convention. But I agree that it would be good to have a convention to use for "value that could be unrestricted or resource". Maybe we should reclaim `Value` for this and be more disciplined about saying `Copyable` or `Unrestricted` when we mean "non-resource value"?